_posts/en/newsletters/2022-01-12-newsletter.md

```diff
@@ -0,0 +1,185 @@
+---
+title: 'Bitcoin Optech Newsletter #182'
+permalink: /en/newsletters/2022/01/12/
+name: 2022-01-12-newsletter
+slug: 2022-01-12-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes an idea to add accounts to Bitcoin for
+paying transaction fees and includes our regular sections with the
+summary of a Bitcoin Core PR Review Club meeting, announcements of new
+releases and release candidates, and descriptions of notable changes to
+popular Bitcoin infrastructure projects.
+
+## News
+
+- **Fee accounts:** Jeremy Rubin [posted][rubin feea] to the Bitcoin-Dev
+  mailing list the rough idea for a soft fork that could make it easier
+  to add fees to presigned transactions, such as those used in LN and
+  other contract protocols.  The idea is an outgrowth of his transaction
+  sponsorship idea described in [Newsletter #116][news116 sponsorship].
+
+    The basic idea for fee accounts is that users could create
+    transactions that deposited bitcoins into an account tracked by
+    upgraded full nodes that understood the new consensus rules.  When
+    the user subsequently wanted to add fees to a transaction, they
+    would sign a short message containing the amount they wanted to pay
+    plus the txid of that transaction.  Upgraded full nodes would allow
+    any block containing both the transaction and the signed message to
+    pay the miner of that block the signed fee amount.
+
+    Rubin suggests that this would eliminate many problems with
+    [CPFP][topic cpfp] and [RBF][topic rbf] fee bumping related to
+    contract protocols where two or more users shared ownership of a
+    UTXO, or other cases where the use of presigned transactions meant
+    the current network feerates couldn't have been known when the
+    transaction was signed in the past.
```

Comment:
I think I'm going to skip this for three reasons: (1) I think it would complicate the description, (2) the old rule that, if you don't make privacy the default, nobody will use it, and (3) I'm lazy.  Also, of course, anyone can read Rubin's original description, like you did, if they want details.